CVInHtml/cv.html

```diff
@@ -132,5 +151,6 @@ <h2>Contact</h2>
             Copyright &copy; 2021
         </div>
     </footer>
+    <script src="bootstrap/js/bootstrap.bundle.min.js"></script>
```

Comment:
The physical file is missing

Reply:
The file bootstrap.min.js has been added as suggested

Reply:
The file source has also been changed from bootstrap/js/bootstrap.bundle.min.js to bootstrap/bootstrap.bundle.min.js